src/Uno.Foundation/AsyncOperationWithProgress.T.cs

```diff
@@ -0,0 +1,46 @@
+﻿#nullable enable
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno;
+
+namespace Windows.Foundation
+{
+	internal class AsyncOperationWithProgress<TResult, TProgress> : AsyncOperation<TResult>, IAsyncOperationWithProgress<TResult, TProgress>, IAsyncOperationWithProgressInternal<TResult, TProgress>
```

Comment:
This class in `internal`, but you mixed the accessibility of members. Some are `internal` and others are `public`.